app/Mails/SendIntegrationActivationReminderEmail.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mails;
+
+use App\Domain\Contacts\Contact;
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Domain\Mail\MailManager;
+use Carbon\Carbon;
+use Illuminate\Console\OutputStyle;
+use Illuminate\Support\Collection;
+use Psr\Log\LoggerInterface;
+
+final readonly class SendIntegrationActivationReminderEmail
```

Comment:
Shouldn't this be a task to execute on the queue? Otherwise, the command to send all the mails will block per mail

Reply:
Now that I have checked the existing implementation, I don't think the `MailManager` gets queued. It does get triggered by an event but it is missing the specific traits for queueing like `use Queueable;`

Reply:
Made them Queueable, also made a ticket for infra to configure the cronjobs once my code is delivered. https://jira.publiq.be/browse/PPF-601